src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -72,6 +72,8 @@ public OpenTelemetryLoggerProvider()
 
         internal OpenTelemetryLoggerProvider(OpenTelemetryLoggerOptions options, IServiceProvider? serviceProvider, bool ownsServiceProvider)
         {
+            OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent("Building OpenTelemetryLoggerProvider.");
```

Comment:
This event is specifically saying MeterProviderSdk event, so it'll be incorrect to use it as is for LoggerProvider. Either create a separate event, or parameterize the provider.